<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotManagementPanel</class>
 <widget class="QWidget" name="ScreenshotManagementPanel">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="list">
     <property name="toolTip">
      <string>All screenshots taken by you are listed here. You can take screenshots by clicking the &quot;Screenshot&quot; item in the context menu of a computer. The screenshots can be managed using the buttons below.</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="previewLbl">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="_2" columnstretch="0,1">
     <item row="1" column="1">
      <widget class="QLabel" name="dateLbl"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="userDescLbl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="userLbl"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="timeLbl"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="hostDescLbl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Computer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dateDescLbl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="hostLbl"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="timeDescLbl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="showBtn">
       <property name="text">
        <string>Show</string>
       </property>
       <property name="icon">
        <iconset resource="../../core/resources/core.qrc">
         <normaloff>:/core/edit-find.png</normaloff>:/core/edit-find.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBtn">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../core/resources/core.qrc">
         <normaloff>:/core/edit-delete.png</normaloff>:/core/edit-delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../core/resources/core.qrc"/>
 </resources>
 <connections/>
</ui>
